drivers/timers/Kconfig

```diff
@@ -479,18 +479,18 @@ config WATCHDOG_AUTOMONITOR_BY_IDLE
 
 endchoice
 
-config WATCHDOG_AUTOMONITOR_TIMEOUT
```

Comment:
should we keep the name as before?

Reply:
If I keep the original name, will it mislead existing developers? I agree with both options.

Reply:
but the unit of WATCHDOG_AUTOMONITOR_PING_INTERVAL is also changed in this patch, it's more confuse that one appends _MSEC suffix, another not

Reply:
ok, let us keep the original name